src/main.ts

```diff
@@ -444,6 +445,16 @@ export class QueryFeature extends SqlOpsFeature<undefined> {
 			);
 		};
 
+		let changeConnectionUri = (newOwnerUri: string, originalOwnerUri: string): Thenable<void> => {
```

Comment:
With the new notification name this would probably be better as `notifyConnectionUriChanged`.

It does seem a little weird to have this be on the public API like this though. There really shouldn't be any reason why customers would ever be calling this (and if they did it would just break a lot of stuff since ADS wouldn't be updated). 

I don't really have a better solution for now without spending more time looking at it, so as long as we keep it in proposed though I guess I'm ok with this for the time being.